client/src/actions/team.js

```diff
@@ -270,17 +271,40 @@ function teamDataAllocator(teamData) {
  *
  * @param {*} teamId id of the team
  * @param {*} teamData data object of the data to be patched
+ * @param {*} successMsg the success message to display if different from default one
  * @returns
  */
-export const updateTeam = (teamId, teamData) => async (dispatch) => {
+export const updateTeam = (teamId, teamData,successMsg = 'Team has been updated') => async (dispatch) => {
```

Comment:
What's the reason for setting the value of successMsg here instead of passing it in?

Reply:
It's to provide a default as the prev implementation did not take it as an arg and need to be backward compatible